src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/Rules/ParameterNamesCannotChangeTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+using Microsoft.DotNet.ApiCompatibility.Tests;
+using Xunit;
+
+namespace Microsoft.DotNet.ApiCompatibility.Rules.Tests
+{
+    public class ParameterNamesCannotChangeTests
+    {
+        public static TheoryData<string, string, CompatDifference[]> TestCases => new()
+        {
+            // Method
+            {
+                 @"
+namespace CompatTests
+{
+  public class First {
+    public void F(int a, string s) {}
```

Comment:
Would it be possible to add tests now that diff parameters that became nullable, or that are decorated with nullability attributes? Or are those not yet supported?

Reply:
I think nullability attributes would be covered by another rule that needs to be added. This rule only cares about parameter names.

Reply:
Nullable operators are filtered out in all existing rules: https://github.com/dotnet/sdk/blob/1b6da85abc592947ca0bb4c7de01d106466258da/src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Extensions/SymbolExtensions.cs#L26.

That's because we will add a specific rule that checks for nullability.